File: app/model/Comment.js
con = require("../../config/connection");


module.exports = {
    CreateComment: function (comment, callback) {
        
        con.query(
            "INSERT INTO phone_indicator.comment" +
            " (post_id, comment_body,user_id)" +
            " VALUES ('" + comment.post_id + "','" + comment.comment_body +
            "','" + comment.user_id + "')", callback)
    },
    UpdateComment: function (comment, callback) {
        con.query(
            "update phone_indicator.comment set comment_body = '" +
            comment.comment_body + "' where comment_id = " + comment.comment_id,callback
        )
    },
    DeleteComment: function (comment_id, callback) {
        con.query('DELETE FROM  phone_indicator.comment WHERE comment_id = ?', comment_id, callback)
    },
    ShowComment: function (post_id, callback) {
        con.query('SELECT * FROM phone_indicator.comment WHERE post_id = '+ post_id+" ORDER BY RAND()", callback)
    }
}

File: project/models/Comment.py
from project.config.Database import connection 
from flask import  jsonify

class Comment:

    def __init__(self):
        self.conn = connection()
        self.cursor = self.conn
    def add_comment(self,params):

        self.cursor.execute("INSERT INTO mobilaty.comment" +
            " (post_id, comment_body,user_id)" +
            " VALUES ('" + params['post_id'] + "','" + params['comment_body'] +
            "','" + params['user_id'] + "')")
  
    def show_comment(self,params):
        self.cursor.execute('SELECT * FROM mobilaty.comment WHERE post_id = '
                            + params['post_id']+" ORDER BY RAND()")
        r = [dict((self.cursor.description[i][0], value)
              for i, value in enumerate(row)) for row in self.cursor.fetchall()]
        return jsonify({'myCollection' : r})
    def update_comment(self,params):
        self.cursor.execute("update mobilaty.comment set comment_body = '" +
            params['comment_body'] + "' where comment_id = " + params['comment_id'])
        #row = cursor.fetchone()

    def delete_comment(self,params):
        self.cursor = self.conn
        self.cursor.execute('DELETE FROM  mobilaty.comment WHERE comment_id = ?', params['comment_id'])
        #row = cursor.fetchone()